animate/metric.py

```diff
@@ -228,8 +228,13 @@ def set_parameters(self, metric_parameters=None):
         self._metric_parameters.update(mp)
         self._variable_parameters.update(vp)
 
+        mp = self._metric_parameters.copy()
+        if "dm_plex_metric_p" in mp:
+            if mp["dm_plex_metric_p"] == np.inf:
```

Comment:
Could simplify and do this in one line `if "dm_plex_metric_p" in mp and mp["dm_plex_metric_p"] == np.inf:`

Reply:
Addressed in 5f99eb0.